platform-ajax/src/main/java/com/softicar/platform/ajax/document/creation/AjaxDocumentCreationService.java

```diff
@@ -44,6 +44,8 @@ private void buildDocument() {
 		document.getEngine().registerCssResourceLink(AjaxCssFiles.AJAX_STYLE.getResource());
 		document.getEngine().registerCssResourceLink(DomCssFiles.DOM_STYLE.getResource());
 
+		document.getDeferredInitializationController().handleAllAppended();
```

Comment:
Note that we still have a second call to `IDomDeferredInitializationController#handleAllAppended` which is in `DomEventHandlerNodeCaller#call`. That one is used whenever `AjaxDomEventService` handles an event.

I theory, those two calls could be unified to a single one in `AbstractAjaxDocumentService` but that would entail quite some refactoring which is not worth the effort.